CppCoreGuidelines.md

```diff
@@ -2079,6 +2079,13 @@ This is usually a very good thing.
 
 Don't try to make all functions `constexpr`. Most computation is best done at run time.
 
+##### Note
+
+Any API that may eventually become configurable or customizable at runtime
+should not be made `constexpr`. Such customization could not be evaluated by the
```

Comment:
Thanks for taking a stab at this.
Suggestion: s/become configurable or customizable at runtime/depends on the component's runtime configuration/
Otherwise, I like it better --  but I still wonder if this formulation applies to

```
   constexpr double mult(double x, double y) { return x * y; }
```

because of runtime floating point environment configuration. I would rather it does not.